packages/edit-navigation/src/components/menu-editor/block-editor-panel.js

```diff
@@ -17,7 +17,12 @@ import { Button, Panel, PanelBody, Popover } from '@wordpress/components';
 import { useSelect } from '@wordpress/data';
 import { __ } from '@wordpress/i18n';
 
-export default function BlockEditorPanel( { saveBlocks } ) {
+/**
+ * Internal dependencies
+ */
+import MenuDelete from '../menu-delete';
+
+export default function BlockEditorPanel( { onDelete, menuId, saveBlocks } ) {
```

Comment:
Let's rename this `onDelete` prop to `onDeleteMenu` to clarify that it's called when the menu is deleted, not the block editor.